common/controllers/form-wizard.js

```diff
@@ -26,9 +26,10 @@ class FormController extends Controller {
       return res.redirect(err.redirect)
     }
 
-    if (err.code === 'SESSION_TIMEOUT') {
-      return res.render('form-wizard-timeout', {
+    if (err.code === 'SESSION_TIMEOUT' || err.code === 'MISSING_PREREQ') {
+      return res.render('form-wizard-error', {
         journeyBaseUrl: req.baseUrl,
+        contentKey: err.code,
```

Comment:
Maybe we should lowercase this so that all the translation keys remain lowercase.